package.json

```
@@ -44,10 +42,12 @@
         "setup": "pnpm install",
         "start": "pnpm build --stream --filter=coveo-styleguide && pnpm -r start --parallel --filter=\\!react-vapor",
         "test": "pnpm -r test --parallel",
+        "cy:open": "pnpm -r cy:open --stream",
+        "cy:run": "pnpm -r cy:run --stream",
         "build": "pnpm -r build --stream",
         "lintfix": "pnpm -r lintfix --parallel",
         "commit-cli": "git-cz",
-        "reconstruct": "npx rimraf node_modules && npm run setup && echo done"
+        "reconstruct": "npx rimraf node_modules && npx rimraf packages/*/node_modules && npm run setup && echo done"
```

I think you can pass a space delimited list of things to delete to rimraf.

```suggestion
        "reconstruct": "npx rimraf node_modules packages/*/node_modules && npm run setup && echo done"
```